src/tests/ftest/launch.py

```diff
@@ -61,16 +61,23 @@ def __init__(self):
 
         def __del__(self):
             """Destroy a TemporaryDirectory object."""
-            rmtree(self.name)
+            # rmtree(self.name)
```

Comment:
Why is this commented?

Reply:
It shouldn't be commented: 74e335c31511c3cbb7111ef1cca3cf80d467a0d8.